blackbox/docs/configuration/enterprise.txt

```diff
@@ -107,9 +107,11 @@ The meaning of the fields of the are as follows:
   | *Runtime:* ``no``
 
   | The client machine addresses that the client matches, and which are allowed
-  | to authenticate. This field can contain an IP address or a CIDR mask. For
-  | example: ``127.0.0.1`` or ``127.0.0.1/32``. If no address is specified in
-  | the entry, then access to CrateDB is open for all hosts.
+  | to authenticate. This field may contain an IPv4 address, an IPv6 address or
+  | an IPv4 CIDR mask. For example: ``127.0.0.1`` or ``127.0.0.1/32``. It also
+  | may contain the special "_local_" notation which will match both IPv4 and
```

Comment:
shouldn't we use backticks for "_local_" as well?

Reply:
yes, you're right